### DISCUSSION TITLE
Problemas para Criar e Rodar Container no DOCKER - Curso FUNDAMENTOS SQL

### DISCUSSION THREAD

**Discussion by lxxzh** [balta-io/forum#214] [Tira Dúvidas] [answered]
Boa tarde @andrebaltieri , estou realizando o curso de Fundamentos SQL e fiz todo o processo de configuração de TERMINAL, WSL e DOCKER. Mas no DOCKER quando executo o passo a passo da criação do CONTAINER e rodar no browser o mesmo não executa. Poderia me apoiar nessa situação? 

<img width="950" alt="image" src="https://user-images.githubusercontent.com/106619427/211406932-9dd4d53a-8e53-47d3-8bdb-27e2ed65f0a0.png">

<img width="957" alt="image" src="https://user-images.githubusercontent.com/106619427/211406993-fcdc24f9-a44d-45b7-ad7d-43c374d86bdc.png">

<img width="960" alt="image" src="https://user-images.githubusercontent.com/106619427/211407055-34cfb595-f240-4030-8236-58f06bc2f8a0.png">

<img width="948" alt="image" src="https://user-images.githubusercontent.com/106619427/211407117-33e8bd37-afee-4c3c-b974-a879f4fceb69.png">

<img width="952" alt="image" src="https://user-images.githubusercontent.com/106619427/211407183-dfc7caec-a067-4ae1-98f7-5a3c43934c6b.png">

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @lxxzh não seria por conta do seu container estar parado? É esse mesmo da imagem que mandou?

**Comment by lxxzh** [comment]
Eu tento dar start nele e não roda, no TERMINAL não da nenhum erro, mas no BROWSER não aparece nada. 

<img width="960" alt="image" src="https://user-images.githubusercontent.com/106619427/211709491-2329bcee-7e33-4e29-b0a3-c66b67d0e73f.png">

<img width="955" alt="image" src="https://user-images.githubusercontent.com/106619427/211709603-891141e5-bb29-4395-af24-516a10c1d1e8.png">

**Reply by andrebaltieri** [reply]
Mas ele aparece como "RUNNING" no seu Dashboard do Docker ou quando você executa o comando `docker ps`?
Se o container não estiver rodando, ele não aparece no Browser.

Qual aula de qual curso você está executando?

**Reply by lxxzh** [reply]
Sim, sei que se o Container não estiver rodando não vai mostrar nada no BROWSER.

Segui certo todo o passo a passo do tutorial, conforme nos prints do TERMINAL acima. Estou na etapa de "Rodando o Container" deste tutorial: https://balta.io/blog/docker-instalacao-configuracao-e-primeiros-passos?utm_source=github&utm_medium=2805-repo&utm_campaign=readme. Para depois iniciar o curso de Fundamentos SQL.

O Container não roda, no campo STATUS ele tenta iniciar a aplicação e não vai. Não aparece o "Running" no DOCKER DESKTOP. 

<img width="775" alt="image" src="https://user-images.githubusercontent.com/106619427/211829963-fecdfebd-bac3-4510-9a84-144af712a0f2.png">

**Reply by andrebaltieri** [reply]
Bom dia, @lxxzh, como vai?

Acredito que esteja havendo uma confusão aqui... O container que você está tentando executar não é o do curso de SQL.
O do curso se chama `sqlserver` e roda na porta 1433.

**Reply by lxxzh** [reply]
Boa tarde Balta, tranquilo e contigo? 

Depois da sua última resposta eu segui este tutorial: https://balta.io/blog/sql-server-docker?utm_campaign=recado_comeco_de_curso_-_docker_e_sql_server&utm_medium=email&utm_source=RD+Station

Fiz os comandos na sequencia do tutorial e deu certo, finalmente consegui configurar o Docker e o Azure! Agradeço a orientação. Bora pro curso !! rsrrs

<img width="954" alt="image" src="https://user-images.githubusercontent.com/106619427/212168711-c68fb05b-8352-4685-8679-258b7ca4537a.png">


<img width="765" alt="image" src="https://user-images.githubusercontent.com/106619427/212168781-e91defe0-1fa1-495a-95ee-a5cdc3963393.png">
